api/src/middlewares/services/StaffService/StaffService.ts

```diff
@@ -122,7 +123,8 @@ export class StaffService {
     }
 
     const country = answers.country as string;
-    const emailBody = this.getEmailBody({ fields, payment: paymentViewModel, reference }, template, type);
+    const postalValue = postal ? "Postal" : "";
+    const emailBody = this.getEmailBody({ fields, payment: paymentViewModel, reference, postal: postalValue }, template, type);
```

Comment:
I think just send `postal` (the boolean) straight through, and use an if statement to select the remapper rather than constructing a string. or construct the string inside `getEmailBody`. I think it gets confusing bc  it's treated as a boolean elsewhere.